docs/index.md

```diff
@@ -41,6 +37,7 @@ ROCm documentation is organized into the following categories:
 :::{grid-item-card} How to
 :class-body: rocm-card-banner rocm-hue-12
 
+* {doc}`Programming guide<hip:index>`
```

Comment:
I don't know if we need this link duplicated here and in the text above, but if we do they should be called the same "HIP programming guide"

Reply:
I would remove it from the box. 
It's already not in the TOC

Reply:
Removed.